src/meal.js

```diff
@@ -0,0 +1,185 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable radix */
+/* eslint-disable class-methods-use-this */
+/* eslint-disable array-callback-return */
```

Comment:
- [ ] It looks like you're using some disablers here, it's ok if we're struggling with some issues and then we use it to continue 1 or 2 times, please, try to do the code without disabling a lot of functions here. 🌵